projects/ccf-rui/src/app/modules/info/info-dialog/info-dialog.component.scss

```diff
@@ -34,5 +34,13 @@
       margin-top: 0;
       margin-bottom: 0.5rem;
     }
+
+    .panel {
+      margin-bottom: 1.5rem;
+    }
+
+    .dialog-content {
+      height: 50rem !important; min-height: 20px !important;
```

Comment:
one line per style please. Also, is `!important` required here? If there is another way that's preferred.